README.md

```diff
@@ -177,6 +177,7 @@ and `{}` for a mutually exclusive keyword.
 | Show DML Execution Plan | `EXPLAIN {INSERT\|UPDATE\|DELETE} ...;` | |
 | Show Query Execution Plan with Stats | `EXPLAIN ANALYZE SELECT ...;` | |
 | Show DML Execution Plan with Stats | `EXPLAIN ANALYZE {INSERT\|UPDATE\|DELETE} ...;` | |
+| Start a new query optimizer statistics package construction | `ANALYZE` | |
```

Comment:
nit: To keep consistent with other statements, please add `;` in the end: `ANALYZE;`

Reply:
Your point sure makes sense. I have updated Syntax doc.

Reply:
Thanks!